src/Persistent_cohomology/include/gudhi/Persistent_cohomology.h

```diff
@@ -159,7 +166,7 @@ class Persistent_cohomology {
       min_length_ = new_length;
     }
 
-    Complex_ds * cpx_;
+    FilteredComplex * cpx_;
```

Comment:
One drawback of just using the template parameter without providing a public typedef for it is that, given a PCOH type, I cannot write any `PCOH::something` to ask what the type of the filtered complex was. It probably doesn't matter, I can't think of a situation where we would need that anyway.